### PR TITLE
Zookeeper latency metrics lack unit resolution

### PR DESCRIPTION
**Pull request by ShubhamRwt**
### Type of change

_Select the type of your PR_

- Bugfix

### Description

This PR solves the issue corresponding to #1548 

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md
- [x] Supply screenshots for visual changes, such as Grafana dashboards

image

### CODE REVIEW

File: examples/metrics/grafana-dashboards/strimzi-zookeeper.json [hunk 1]
```diff
@@ -1165,7 +1165,7 @@
       "dashLength": 10,
       "dashes": false,
       "datasource": "${DS_PROMETHEUS}",
-      "description": "Amount of time it takes for the server to respond to a client request",
+      "description": "Amount of time(in ms) it takes for the server to respond to a client request",
```
Comment (tombentley) [inline examples/metrics/grafana-dashboards/strimzi-zookeeper.json:1168]:
```suggestion
      "description": "Amount of time (in ms) it takes for the server to respond to a client request",
```

File: examples/metrics/grafana-dashboards/strimzi-zookeeper.json [hunk 2]
```diff
@@ -1250,7 +1250,7 @@
...
```
Comment (tombentley) [inline examples/metrics/grafana-dashboards/strimzi-zookeeper.json:1253]:
```suggestion
      "description": "Amount of time (in ms) it takes for the server to respond to a client request",
```

File: examples/metrics/grafana-dashboards/strimzi-zookeeper.json [hunk 3]
```diff
@@ -1335,7 +1335,7 @@
...
```
Comment (tombentley) [inline examples/metrics/grafana-dashboards/strimzi-zookeeper.json:1338]:
```suggestion
      "description": "Amount of time (in ms) it takes for the server to respond to a client request",
```

File: examples/metrics/grafana-dashboards/strimzi-zookeeper.json [hunk 4]
```diff
@@ -1223,8 +1223,8 @@
       },
       "yaxes": [
         {
-          "format": "short",
-          "label": null,
+          "format": "ms",
+          "label": "ms",
```
Comment (tombentley) [inline examples/metrics/grafana-dashboards/strimzi-zookeeper.json:1227]: Should the label be more like "request latency (ms)"?

**Review by ppatierno — APPROVED** [review]

**Comment by ShubhamRwt** [comment]
@tombentley should we make the label like "Request Latency (in ms)" ?

**Review by tombentley — APPROVED** [review]
Thanks!

**Pull request merged**